app/views/errors/unauthorised.html.haml

```diff
@@ -0,0 +1,4 @@
+- content_for :page_title_prefix, t('error_pages.unauthorised')
+
+%h1.heading-xlarge=t('error_pages.unauthorised')
+%p You can email #{mail_to t('help.email')} if the problem persists.
```

Comment:
This can probably be extracted to the locale and reused between this template & the internal_server_error one

Reply:
I was being consistent with the other error pages and did not want to take this opportunity to address them all incase those changes got locked into this more complex change.